README.md

```diff
@@ -1,92 +1,74 @@
   </a>
+</br>
+</br>
+
+[![Sync Mainnet](https://github.com/nimiq/validators-api/actions/workflows/sync-mainnet.yml/badge.svg)](https://github.com/nimiq/validators-api/actions/workflows/sync-mainnet.yml) [![Sync Testnet](https://github.com/nimiq/validators-api/actions/workflows/sync-testnet.yml/badge.svg)](https://github.com/nimiq/validators-api/actions/workflows/sync-testnet.yml)
 
-<p align="center">
-  Details of validators in the Nimiq Blockchain and their scores, calculated using Nimiq's VTS algorithm.
-<p>
+The Nimiq Validators API enables staking pools, based on single validators, to integrate with the Nimiq Wallet. This helps stakers make informed decisions when choosing where to stake their funds.
 
-[![Sync Mainnet](https://github.com/nimiq/validators-api/actions/workflows/sync-mainnet.yml/badge.svg)](https://github.com/nimiq/validators-api/actions/workflows/sync-mainnet.yml)
-[![Sync Testnet](https://github.com/nimiq/validators-api/actions/workflows/sync-testnet.yml/badge.svg)](https://github.com/nimiq/validators-api/actions/workflows/sync-testnet.yml)
+**Validators and Staking Pools**:
+A validator can operate as a staking pool, allowing multiple users to stake. Pools must provide detailed information such as fees, payout schedules, and contact details to ensure trust and transparency.
 
-## Add your validator information
+Stakers can evaluate a validator’s reliability using the Validator Trust Score (VTS) and review staking pool details like payout schedules to select the best option.
 
-If you run your own validator and want it to be recognized, you can add a name, logo and other publicly available fields. For example, the Nimiq Wallet will use this information to display your validator to users.
+## Add your Validator Information
 
-In order to do so, follow these steps:
+If you operate a staking pool and want to be displayed in the Nimiq Wallet, follow these steps:
 
 1. [Fork this repository](https://github.com/nimiq/validators-api/fork).
-2. Create a new file in the [validators folder](./public/validators/main-albatross).
-3. Read the [Recommendations for Writing Your Validator Information](#recommendations-for-your-validator-information)
-4. Learn about the [JSON schema](#validator-json-schema)
-5. Submit a PR to this repo. Someone in the team will review it.
-6. Once the PR is submitted, check that the [API endpoint](https://validators-api-mainnet.nuxt.dev/api/v1) is returning your information. It can take a few minutes.
+2. Add your Validator File:
+   - Create a new JSON file in the `public/validators/main-albatross` directory.
+   - Use the provided example template in the directory to structure your data.
+3. Review the [Description Guidelines](#recommendations-for-your-validator-information).
+4. Learn about the [JSON Schema](#validator-json-schema).
+5. Submit a PR to this repository. A Nimiq team member will review your submission.
```

Comment:
At the moment this process is quite opaque and slow (up to 3 days).

Can we add something about this? We need to be more transparent:

- Why it takes so much?
- What are we verifying exactly?


1. We need to verify the website contents
2. We need to verify that the fee stated by the validator is what actually is doing (not sure how this is being tested, ask Micha)...

Not sure if this is the place